charmcraft.yaml

```diff
@@ -175,3 +175,8 @@ config:
       type: string
       default: info
       description: Log level for the UPF. One of `debug`, `info`, `warn`, `error`, `fatal`, `panic`.
+    bess-http:
+      type: boolean
+      default: false
+      description: |
+        When enabled, UPF will expose the BESS HTTP server on port 5000
```

Comment:
I would add something in the vein of:

```
WARNING: This should be enabled only for debugging and disabled afterwards.
```

I want to make it clear that it's not just a nice feature that one enables on their production deployment.